src/API/HealthController.ts

```diff
@@ -0,0 +1,28 @@
+import { inject, injectable } from 'inversify'
+import { Db } from 'mongodb'
+
+import { IPFS } from './IPFS'
+
+@injectable()
+export class HealthController {
+  private readonly db: Db
+  private readonly ipfs: IPFS
+
+  constructor(@inject('DB') db: Db, @inject('IPFS') ipfs: IPFS) {
+    this.db = db
+    this.ipfs = ipfs
+  }
+
+  async getHealth(): Promise<any> {
+    const mongoConnection = await this.db.stats()
+    const mongoIsConnected = mongoConnection.ok === 1
```

Comment:
make this logic `mongoConnection.ok === 1` a function and unit test it. The function can be external of the class so that we don't' need to instantiate the class.

Reply:
does `db.stats()` never throw? if the connection to the db is broken completely, does it return an object anyway?

Reply:
good question, will leave to you @jwicks31

Reply:
The [Mongo Docs](https://docs.mongodb.com/manual/reference/method/db.stats/#accuracy-after-unexpected-shutdown) make it seem to me like it will return an object. 

I've tried testing manually by giving incorrect ENV variables for Mongo, but the app crashes. So I wasn't able to verify what is returned.

 I also tested by closing the MongoClient right after starting it inside of the API module. This did throw.

Reply:
Thanks @jwicks31! App crashing or throwing works. Errors thrown in API get caught in a middleware and logged, so alarms will get triggered.